src/filter.cpp

```diff
@@ -1040,7 +1048,7 @@ Option<bool> filter::parse_filter_string(const std::string& filter_query, std::s
         } else if (preceding_colon && c != ' ') {
             preceding_colon = false;
         }
-    } while (index < size && (inBacktick || is_geo_value ||
+    } while (index < size && (inBacktick || inQuotes || is_geo_value ||
```

Comment:
I think we wouldn't need to add `inQuotes` logic in `filter::parse_filter_string` because we would've already removed the enclosing filter value quotes in `toFilter()`.

Reply:
Removed thanks.